lang/pl.json

```diff
@@ -67,13 +67,34 @@
     "Open main menu": "Otwórz menu główne",
     "Please verify your credentials and try again": "Sprawdź swoje dane logowania i spróbuj ponownie",
     "Invalid password or username": "Nieprawidłowe hasło lub nazwa użytkownika",
-
+    "Show favorites": "Pokaż ulubione",
+    "City added to favorites!": "Miasto dodane do ulubionych!",
```

Comment:
I think I will remove exclamation mark at the end of the sentence in all toast notifications.

Reply:
fixed